ngraph_bridge/ngraph_tensor_manager.cc

```diff
@@ -63,6 +63,33 @@ void NGraphTensorManager::Initialize() {
       FindComplement(m_number_of_inputs, m_input_indexes_from_variables);
   m_pipelined_output_indexes =
       FindComplement(m_number_of_outputs, m_output_indexes_assigning_variable);
+
+  string node_prefetch_key =
+      NGraphCatalog::CreateNodeKey(m_ng_encap_graph_id, m_ng_encap_node_name);
```

Comment:
I would rather not have the user create the key. So I will add utility for NGraphCatalog::ExistsInPrefetchedInputIndexMap(graph_id, name)

Reply:
The utility already exists https://github.com/tensorflow/ngraph-bridge/blob/master/ngraph_bridge/ngraph_catalog.h#L154